tests/src/main/java/org/hibernate/beanvalidation/tck/tests/validation/groupconversion/RegisteredAddresses.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Bean Validation TCK
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.beanvalidation.tck.tests.validation.groupconversion;
+
+import java.time.Year;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.validation.Valid;
+import javax.validation.groups.ConvertGroup;
+import javax.validation.groups.Default;
+
+/**
+ * @author Guillaume Smet
+ */
+public class RegisteredAddresses {
+
+	private final Map<Year, List<@Valid @ConvertGroup(from = Default.class, to = BasicPostal.class) Address>> mainAddresses = new HashMap<>();
+
+	private final Map<Year, List<Address>> shipmentAddresses = new HashMap<>();
+
+	private final Map<Year, List<@Valid @ConvertGroup.List({
```

Comment:
Not really. It came from another test and I let it this way hoping that it would avoid triggering the repeatable issue with Eclipse but it didn't work either. So we can probably remove them.